ctcusc/RiverLA
# Added an Integration Test

## Ticket Information

**Link to Notion**

**Short Description**
Added an integration test which tests the flow of creating a new volunteer on RiverLa's website.
The integration test takes in an email from the user as console input and prints out instructions of how to check the test result to the console.
## Manual Test Plan
To see if the test passed, you must manually go to NationBuilder's website and see if a person was created with the specified email address. You also must login to this specified email address to check if a new volunteer was created. 
## PR Checklist

- [] Wrote ava tests
- [ ] Reviewed by **all** @ctcusc/RiverLA team members
